lldb/include/lldb/Target/Language.h

```diff
@@ -379,6 +380,21 @@ class Language : public PluginInterface {
   /// Python uses \b except. Defaults to \b catch.
   virtual llvm::StringRef GetCatchKeyword() const { return "catch"; }
 
+  /// Method invoked by \a Module::LookupInfo::Prune used to filter out symbol
+  /// search results.
+  ///
+  /// \param[in] sc A symbol that passed the common symbol search lookup
+  ///   process.
+  /// \param[in] lookup_info The full lookup info.
+  ///
+  /// \return whether the given symbol should be discarded from the search
+  /// results.
+  virtual bool
+  SymbolLookupHookShouldPruneResult(const SymbolContext &sc,
```

Comment:
I don't think "Hook" is right here.  You can tell this is a language hook by the fact that we're calling some GetLanguagePlugin->ThisMethod so the "Hook" is redundant.  We don't call any of the other language hooks hooks.